tests/Examples/tfhe_rust/add_one.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: heir-opt --forward-store-to-load --cggi-to-tfhe-rust --canonicalize --cse %s | heir-translate --emit-tfhe-rust > %S/src/fn_under_test.rs
+// RUN: heir-opt --forward-store-to-load --cggi-to-tfhe-rust --canonicalize --cse %s | heir-translate --emit-tfhe-rust > %S/src/add_one_test_rs_lib.rs
 // RUN: cargo run --release --manifest-path %S/Cargo.toml --bin main_add_one -- 2 --message_bits=3 | FileCheck %s
```

Comment:
Should we remove this RUN line? Or we're just not running these through lit anymore at all, so it doesn't matter? Maybe we can remove the lit directives entirely. Same for the other files.